src/pages/events/[slug].astro

```diff
@@ -151,68 +123,63 @@ const team = [
+          <p set:html={event.content_html} />
         </div>
 
         <Button
           text="Jetzt Anmelden"
           color="dark"
           class="mt-12 uppercase lg:mt-8"
-          href="https://pretix.eu/yv/Forschungsraum"
+          href={event.registrationLink}
         />
       </div>
     </div>
 
-    <div class="mx-auto mt-6 max-w-[80rem] pt-12 lg:mt-14 lg:text-xl">
-      <Image src={Timetable} class="w-full object-cover" alt="Wochenplan Bauwoche 2024" />
-    </div>
+    {
+      event.timetable && (
+        <div class="mx-auto mt-6 max-w-[80rem] pt-12 lg:mt-14 lg:text-xl">
+          <img
+            src={event.timetable.src.src}
+            class="w-full object-cover"
+            alt={event.timetable.alt}
+          />
+        </div>
+      )
+    }
     <div class="mx-auto mt-6 max-w-[80rem] px-4 lg:mt-12 lg:px-24">
-      <div>
-        <h2 class="font-serif text-2xl font-bold lg:text-4xl">Begleitung und Referierende</h2>
-        <div class="-mt-4 grid grid-cols-1 gap-x-6 lg:-mt-6 lg:grid-cols-2 lg:gap-x-12">
-          {
-            team.map((member) => (
-              <div class="row-span-3 mt-8 grid grid-rows-subgrid lg:mt-16">
-                <Image
-                  src={member.image}
-                  alt=""
-                  class="aspect-square rounded-lg object-cover"
-                  sizes="(max-width: 1024px) 100vw, 440px"
-                  widths={[360, 440, 720, 880, 1024]}
-                />
-                <div class="mt-2 lg:mt-4">
-                  <p class="text-xl font-bold lg:text-2xl">{member.name}</p>
-                  <p class="text-sm text-gray-700">{member.role}</p>
+      {
+        event.team?.length != 0 && (
+          <div>
+            <h2 class="font-serif text-2xl font-bold lg:text-4xl">Begleitung und Referierende</h2>
+            <div class="-mt-4 grid grid-cols-1 gap-x-6 lg:-mt-6 lg:grid-cols-2 lg:gap-x-12">
+              {event.team?.map(async (member) => (
+                <div class="row-span-3 mt-8 grid grid-rows-subgrid lg:mt-16">
+                  <img
+                    src={member.image.src.src}
+                    alt=""
+                    class="aspect-square rounded-lg object-cover"
+                    sizes="(max-width: 1024px) 100vw, 440px"
+                  />
+                  <div class="mt-2 lg:mt-4">
+                    <p class="text-xl font-bold lg:text-2xl">{member.name}</p>
+                    <p class="text-sm text-gray-700">{member.job}</p>
+                  </div>
+                  <p class="mt-4 text-justify lg:mt-6">{member.bio}</p>
                 </div>
-                <p class="mt-4 text-justify lg:mt-6">{member.description}</p>
-              </div>
-            ))
-          }
-        </div>
-      </div>
+              ))}
+            </div>
+          </div>
+        )
+      }
       <div class="py-12">
         <h2 class="font-serif text-2xl font-bold lg:text-4xl">Veranstaltungsort Rosow</h2>
```

Comment:
This section only makes sense for events that actually are in Rosow

Reply:
Hm yeah, maybe we do it like this for now and then add a field / relation in the CMS later? This year all our events will be in Rosow I believe.